README.md

```diff
@@ -37,17 +37,30 @@ yarn run jasmine:ci
 
 ### Further documentation
 
+Using the gem
```

Comment:
Do you think that this should be a heading?

```suggestion
#### Using the gem
```

Reply:
Or add a colon at the end:
```suggestion
Using the gem:
```

Reply:
Have updated the headings here a bit based on this suggestion, hopefully a bit clearer now.